app/ante/vesting.go

```diff
@@ -80,11 +87,33 @@ func (vtd EthVestingTransactionDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx,
 			)
 		}
 
-		// Error if account has locked coins (before surpassing all lockup periods)
-		islocked := clawbackAccount.HasLockedCoins(ctx.BlockTime())
-		if islocked {
-			return ctx, errorsmod.Wrapf(vestingtypes.ErrVestingLockup,
-				"cannot perform Ethereum tx with clawback vesting account, that has locked coins: %s", vested,
+		// Check to make sure that the account does not exceed its spendable balances.
+		// This transaction would fail in processing, so we should prevent it from
+		// moving past the AnteHandler.
+		msgValue := sdk.NewCoin(types.BaseDenom, math.NewIntFromBigInt(msgEthTx.AsTransaction().Value()))
```

Comment:
this is the wrong denom, use the one from the EVM module params

Reply:
also, why are you converting to tx? seems unnecessary

Reply:
From what I can tell, there's no interface to get the `value` field other than converting to a Tx. Is there a more efficient way to do this?